domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/TimelineEventTest.java

```diff
@@ -21,7 +21,7 @@ public void shouldBeSuccessfulValidationForCorrectTimelineEvent() {
 
     @Test
     public void shouldFailValidationForNullEventDate() {
-        TimelineEvent timelineEvent = new TimelineEvent(null, "description");
+        TimelineEvent timelineEvent = TimelineEvent.builder().eventDate(null).description("description").build();
```

Comment:
just do builder().description("ddddd").build(). no need for eventDate to set explicitly as null

Reply:
done